onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1132,6 +1154,71 @@ def _should_skip_test_sample(op_name: str, sample) -> Optional[str]:
     return None
 
 
+class OrtAbortedError(RuntimeError):
+    """ONNX Runtime Aborted."""
+
+
+def _ort_session_run(serialized_model, ort_inputs, return_dict) -> None:
+    """Run a model with ONNX Runtime and store the results in return_dict."""
+
+    # Handle SIGSEGV so Python does not abort
+    def sig_handler(signum, frame):
+        warnings.warn(f"Segmentation error ({signum}) detected in ORT:\n{frame}")
+
+    signal.signal(signal.SIGSEGV, sig_handler)
```

Comment:
do we need to reset this somewhere?

Reply:
Good question. I am not sure. Let me check

Reply:
I removed them